CommunicationLayer/CommunicationContainer.cpp

```diff
@@ -2,12 +2,9 @@
 #include <QUdpSocket>
 
 #include "../DataLayer/DataContainer.h"
-#include "CommunicationContainer.h"
-#include "CommDeviceControl/ConnectionController.h"
-#include "CommDeviceControl/CommDeviceManager.h"
```

Comment:
We probably still need this, we will be connecting using `UdpConnectionService` later. I can understand removing `UdpConnectionService` itself since it won't be implemented yet. But CommDeviceManager should still be connecting the serial connection.